src/components/PersonalData/Birthday/Birthday.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+
+const Birthday = props => {
+  return (
+    <div>
+      <label htmlFor="exampleInputName">Data Urodzin</label>
+      <br />
+      <DatePicker
+        
+        id="birthday"
+        dateFormat="dd/MM/yyyy"
+        selected={props.birthday}
+        onChange={props.birthdayHandler}
```

Comment:
I think it would be better to call it *birthdayOnChange" or sth like that - it would be more intuitive